tests/test_singularity.py

```diff
@@ -41,11 +41,14 @@ def test_singularity_incorrect_image_pull():
     assert result_code != 0
 
```

Comment:
The below caused all tests to pass but did not increase code coverage. So there now appears to be a test in place to determine that the code does in fact produce an image locally if no env variable for $SINGULARITY_PULLFOLDER is set or in the directory if the variable is set, but it did not increase code coverage. Any ideas?